app/models/transmittable/transaction.rb

```diff
@@ -68,6 +68,7 @@ class Transaction
 
     # Indexes
     index({ key: 1 })
+    index({ transactable_id: 1 })
```

Comment:
https://github.com/ideacrew/crm_gateway/pull/56/files#diff-5a55385eb24f602100826a1002e810ffcc6fb112dcfef029d71e0477015fc4f0R17 should already cover the indexes for `transactable_id` and `transactable_type`. Please verify.